drivers/timer/Kconfig

```diff
@@ -165,6 +165,25 @@ config NRF_RTC_TIMER
 	  Counter NRF_RTC1 and provides the standard "system clock driver"
 	  interfaces.
 
+config STM32_LPTIM_TIMER
+	bool "STM32 Low Power Timer"
+	depends on SOC_FAMILY_STM32
+	depends on CLOCK_CONTROL
+	select TICKLESS_CAPABLE
+	help
+	  This module implements a kernel device driver for the LowPower Timer
+	  and provides the standard "system clock driver" interfaces.
+
+config LPTIM_CLOCK_LSI
```

Comment:
Since it's one or the other, use a choice semantic

Reply:
with choice on STM32_LPTIM_CLOCK
STM32_LPTIM_CLOCK_LSI or STM32_LPTIM_CLOCK_LSE in the Kconfig - timer driver configuration options